sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -772,6 +777,11 @@
                             "tableView": false,
                             "modalEdit": false,
                             "values": [
+                                {
+                                    "label": "COVID-19",
+                                    "value": "covid19",
+                                    "shortcut": ""
+                                },
                                 {
```

Comment:
Are the changes on top of latest code from main? the Covid-19 was removed in one of the recent efforts by @guru-aot in #3375